taxonomies/food/ingredients.txt

```diff
@@ -2410,20 +2410,12 @@ wikipedia:fr: https://fr.wikipedia.org/wiki/Beurre_doux
 en: 82% fat unsalted Butter
 fr: Beurre à 82% MG doux
 hr: neslani maslac 82% masti
-it: Burro non salato 82% di grassi, burro dolce 82% di grassi
-ciqual_food_code:en: 16400
-ciqual_food_name:en: Butter, 82% fat, unsalted
-ciqual_food_name:fr: Beurre à 82% MG, doux
-
-< en:unsalted butter
-en: 82% fat unsalted butter
-fr: Beurre à 82% MG doux
-hr: neslani maslac 82% masti
-it: Burro morbido non salato 82% di grassi, burro morbido dolce 82% di grassi
+it: Burro non salato 82% di grassi, burro dolce 82% di grassi, Burro morbido non salato 82% di grassi, burro morbido dolce 82% di grassi
 ciqual_food_code:en: 16404
 ciqual_food_name:en: Butter, 82% fat, unsalted, easy-to-spread
 ciqual_food_name:fr: Beurre à 82% MG, doux, tendre
```

Comment:
@aleene seems like there was a difference in Ciqual food code, I don't know which we should keep, or if we should have an entry for  *82% fat unsalted easy to spalm butter*.